### ISSUE TITLE
Support for Compose Preview Screenshot Testing from Google

### ISSUE THREAD

**Issue by davidvavra** [airbnb/Showkase#387]
Hi there,
Google announced official support for screenshot testing on I/O:

https://developer.android.com/studio/preview/compose-screenshot-testing

It looks great, but it doesn't support automatic collection of all previews - you need to explicitly put them into a test class. So Showcase would be helpful. Any plans to create a sample, similar to Paparazzi and Shot samples?

**Comment by vinaygaba** [comment]
Yes, would be a great way to overcome the current limitations of manually adding the previews.

**Comment by davidvavra** [comment]
Hi I had a chat with Jose Alcerreca on I/O Connect event, he is a Googler responsible for the tool. He said that there is no API like Paparazzi has for taking screenshots. The possible way would be to generate code for the previews in screenshotTest source set. Can you guide me a bit how could I do that with Showkase? I might have time to do it then.
